Automattic/wp-calypso
# Update fetchCurrentTheme, with proper naming and tests.

## Intoduction
Aim of this PR is to update fetchCurrentTheme action with naming that matches rest of Calypso and also adding unit tests that will help with further themes Redux subtree refactor.

3 Redux state tests are added in this PR:
- fetching current theme action dispatch
- success action dispatch
- failure action dispatch

To test:
1. open `/design` in single site mode
2. check if current theme name in properly displayed in current theme banner at the top of the page.

![current-theme](https://cloud.githubusercontent.com/assets/17271089/20387318/c8850300-acbf-11e6-8038-b26660a1a544.png)
